docs/reference/ci-cd.md

```diff
@@ -0,0 +1,10 @@
+# Our CI/CD system
+
+The best place to look to learn about the latest state of our *automatic* hub deployment
+is to look at [the `deploy-hubs.yaml` GitHub workflow](https://github.com/2i2c-org/pilot-hubs/blob/master/.github/workflows/deploy-hubs.yaml).
+That workflow defines the automatic hub deployment for many of our major clusters.
+
+:::{note}
+This is essentially a placeholder, it needs to be extended to talk a **lot** more about
+our CI/CD system.
```

Comment:
Maybe make it even more visible that we need to write docs about our deployment automation :sweat_smile: 

```suggestion
TODO: This is essentially a placeholder, it needs to be extended to talk a **lot** more about
our CI/CD system.
```